src/cmd/backup-restore-tachyon.js

```diff
@@ -96,6 +100,8 @@ module.exports = class BackupRestoreTachyonCommand extends CLICommandBase {
 	async restore({
 		'input-dir': inputDir = process.cwd(),
 		'log-dir': logDir = this._logsDir,
+		'force-cloud': forceCloud,
+		filePath,
```

Comment:
the command line argument is lowercase `filepath`

Reply:
fixed here! https://github.com/particle-iot/particle-cli/pull/867/commits/d3719484072d800d64334c4a96c378d6839fd861